scripts/run-integration-tests

```diff
@@ -50,7 +50,7 @@ PLUGINS_PATH=${PLUGINS_PATH:-''}
 # FILE: file to test
 # OUTPUT_PATH: Directory where results are saved (e.g. "some/path/to/Testfile-cy.js-1")
 function executeCypress {
-  npm run --silent cypress -- \
+  timeout 20m npm run --silent cypress -- \
```

Comment:
This kills the run after 20 minutes?

Reply:
Yes: https://www.gnu.org/software/coreutils/manual/html_node/timeout-invocation.html